www/js/controllers/database-controller.js

```diff
@@ -0,0 +1,14 @@
+module.controller('MyCtrl', function($scope, $cordovaSQLite) {
```

Comment:
Cool, just make sure that you name the controller something that matches the name of the file. So I would name it 'DatabaseController' as opposed to 'MyCtrl'